helm/sonar/Chart.yaml

```diff
@@ -1,15 +1,16 @@
-apiVersion: v1
-description: A Helm chart for Kubernetes
+apiVersion: v2
+description: A sonar service helm chart
```

Comment:
A helm chart for the sonar service which is the central point for all monitoring features